.github/workflows/pinned_build.yaml

```diff
@@ -12,10 +12,48 @@ defaults:
     shell: bash
 
 jobs:
-  Temp:
-    name: temp
-    runs-on: ubuntu-latest
+  Build:
+    name: Build
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu18, ubuntu20, ubuntu22]
+    runs-on: ["self-hosted", "enf-x86-beefy-long"]
+    container: ${{ matrix.platform == 'ubuntu18' && 'ubuntu:bionic' || matrix.platform == 'ubuntu20' && 'ubuntu:focal' || 'ubuntu:jammy' }}
```

Comment:
Do we need th ubuntu18 stuff for 4.0?

Reply:
Good call.  Thanks for catching that.  Addressed: [933c550](https://github.com/AntelopeIO/leap/pull/1339/commits/933c550ce90c8b2e0e4eb38d44d1b04466ca1a56)